docs/use_cases.md

```diff
@@ -3,7 +3,61 @@
 Here are examples of questions this tool can help to answer,
 types of results it can give, and other use cases.
 
-## Synthetic F280 records
+## What is the approximate contamination rate of a set of inspected
+consignments?
+
+Given known inspection protocols (sample size and selection method) and
+records containing the inpsection outcomes, what is the approxiamte
+contamination rate of the inspected consignments? By simulating
+inspections using the same protocol and calibrating the contamination
+parameters so that the simulated inspection outcomes match the actual
+inspection outcomes, we can estimate the contamination rate present
+in the actual consignments.
+
+## What is the approximate slippage rate for an inspection protocol?
+
+Given assumed contamination rates and consignment sizes, what percentage
+of contaminates are not being intercepted by border inspections when using
+various inspection methods? This information could be useful for estimating
+propagule pressure for quarantine significant pest species.
```

Comment:
...quarantine **of**?

Reply:
Trying to use APHIS terminology here. "quarantine-significant pest"

Reply:
Makes sense!